firewood/src/merkle/node.rs

```diff
@@ -417,7 +417,14 @@ impl<T> PartialEq for EncodedNode<T> {
 }
 
 #[derive(Debug)]
-struct PathWithBitsPrefix(Path);
+struct PathWithBitsPrefix(u64, Path);
```

Comment:
I actually don't think you need the extra field which makes the diff way smaller